frontend/src/metabase/parameters/components/ParameterSidebar/ParameterSidebar.tsx

```diff
@@ -140,6 +138,14 @@ export const ParameterSidebar = ({
   const handleChangeRequired = (value: boolean) =>
     onChangeRequired(parameterId, value);
 
+  const handleTabChange = (newTab: string | null) => {
+    if (!newTab || (newTab !== "settings" && newTab !== "filters")) {
```

Comment:
IMO typing ids of tabs is an overkill and adds unnecessary complexity, which shows e.g. here.

I'd just go with `string`.

Reply:
I agree, at the same time it's so easy to make a mistake with plain strings, so I decided to type them. Having enum would make things even more complicated